### DISCUSSION TITLE
Best platform for paper organization, reading, annotating, note taking

### DISCUSSION THREAD

**Discussion by sr320** [RobertsLab/resources#1152] [Q&A]
I realize we revisit this often but wanted to see where folks were with this. I am contemplating spending more time in Paperpile to see if it truly fits all my needs - but looking for all options. If you have not found anything ideal - please do go ahead and list features you would like to have.

**Comment by mattgeorgephd** [comment]
I've been playing around with Paperpile and haven't found a good place to add comments/notes; I like the integration with google docs and the ease with which pdfs are grabbed and appended to references within the manager. Traditionally I have used Zotero (as a poor grad student) and more recently Endnote. Zotero is open source and has a lot of bugs. Endnote is polished, but expensive, doesn't grab pdfs automatically, and doesn't support google docs.

**Reply by ksil91** [reply]
Paperpile on Chrome lets you add comments to the PDF, and you can attached docs with notes to the reference using the paper clip icon. I personally love Paperpile.
